ui/src/views/hooks/useTags.ts

```diff
@@ -0,0 +1,22 @@
+import { ApolloError, useMutation } from '@apollo/client'
+import { TagType } from 'src/@types'
+import { UPDATE_TAGS } from 'src/api-logic/graphql/mutations/repos'
+import { showErrorAlert } from 'src/utils/alerts'
+
+const useTags = () => {
+  const [updateTags] = useMutation(UPDATE_TAGS, {
+    onError: (error: ApolloError) => {
+      showErrorAlert(error.message)
+    }
+  })
+
+  const saveTags = (id: string, tags: TagType[]) => {
+    console.log('TAGs: ', tags)
```

Comment:
maybe we should remove this?

Reply:
Done [533ed2f](https://github.com/mergestat/fuse/pull/299/commits/533ed2fe6d6d97b9876eb165a8f533674c9fe48b)